test/parallel/test-signal-args.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+if (common.isWindows) {
+  common.skip('Signal handlers are not supported on Windows');
```

Comment:
a nit: They are somewhat supported. Maybe "using process.kill to send signals is not supported"?

Reply:
Thanks for the suggestion, I updated the comment.